client/lib/features/admin/presentation/views/overview_tab.dart

```diff
@@ -1,464 +1,389 @@
+                            updateProfileImage: (String imageUrl) =>
+                                _updateProfileImage(imageUrl: imageUrl),
+                            removeImage: _removeImage,
+                          ),
+                          SizedBox(height: 30),
+                          ChooseColorSection(
+                            community: _community,
+                            setDarkColor: (val) => _community =
+                                _community.copyWith(themeDarkColor: val),
+                            setLightColor: (val) => _community =
+                                _community.copyWith(themeLightColor: val),
+                          ),
+                          if (mobile) SizedBox(height: 90),
+                        ],
+                      ),
+                    ),
+                    mobile,
+                  ),
+                ],
               ),
-            ],
-          ),
-        ],
-      );
-    } else {
-      return Row(
-        children: [
-          Expanded(
-            child: StepProgressIndicator(
-              completedStepCount: completedStepCount,
-              totalSteps: totalSteps,
             ),
-          ),
-          SizedBox(width: 10),
-          _buildStepsCounter(completedStepCount, totalSteps),
+          );
+        },
+      ),
+      floatingActionButton: ActionButton(
+        onPressed: () => alertOnError(context, _submitFunction),
+        text: context.l10n.save,
+        color: Theme.of(context).primaryColor,
+      ),
+    );
+  }
+
+  Future<void> _updateCommunity() async {
+    await cloudFunctionsCommunityService.updateCommunity(
+      UpdateCommunityRequest(
+        community: _community,
+        keys: [
+          Community.kFieldName,
+          Community.kFieldDisplayIds,
+          Community.kFieldTagLine,
+          Community.kFieldDescription,
+          Community.kFieldContactEmail,
+          Community.kFieldWebsiteUrl,
+          Community.kFieldFacebookUrl,
+          Community.kFieldLinkedinUrl,
+          Community.kFieldTwitterUrl,
+          Community.kFieldBlueskyUrl,
+          Community.kFieldThemeLightColor,
+          Community.kFieldThemeDarkColor,
         ],
+      ),
+    );
+
+    analytics.logEvent(
+      AnalyticsUpdateCommunityMetadataEvent(communityId: _community.id),
+    );
+  }
+
+  bool _verifyContrastOfSelectedTheme() {
+    final light = _community.themeLightColor ?? '';
+    final dark = _community.themeDarkColor ?? '';
+    if (light.isEmpty && dark.isEmpty) return true;
+    if (!ThemeUtils.isColorValid(light)) {
+      _community = _community.copyWith(
+        themeLightColor:
+            ThemeUtils.convertToHexString(context.theme.colorScheme.surface),
       );
+      return false;
+    }
+    if (!ThemeUtils.isColorValid(dark)) {
+      _community = _community.copyWith(
+        themeDarkColor:
+            ThemeUtils.convertToHexString(context.theme.colorScheme.primary),
```

Comment:
I think a bug fix regressed here! I was unable to choose any of the preset color themes until changing this line to `primary` -- this should be what's on staging.
```suggestion
            ThemeUtils.convertToHexString(context.theme.colorScheme.primary),
```

Reply:
OK this comment did not work properly for some reason -- I was referring to this block in `community_theme_utils.dart` on staging: https://github.com/berkmancenter/frankly/blob/staging/client/lib/features/community/utils/community_theme_utils.dart#L98-L102

And is currently `context.theme.colorScheme.secondary` on this branch. I would just double check this section of the code against staging in case there were other merge conflicts!